src/NUnitFramework/framework/Internal/Tests/ParameterizedFixtureSuite.cs

```diff
@@ -42,6 +42,16 @@ public ParameterizedFixtureSuite(ITypeInfo typeInfo) : base(typeInfo.Namespace,
             _genericFixture = typeInfo.ContainsGenericParameters;
         }
 
+        /// <summary>
+        /// Copy constructor style to create a filtered copy of the given parameterized fixture suite
+        /// </summary>
+        /// <param name="suite">Parameterized Fixture Suite to copy</param>
```

Comment:
I'd like to either use `The <see cref="ParameterizedFixtureSuite"/> to copy.` or `The suite to copy.`

(Same comment for all occurrences.)